crates/uv/tests/it/sync.rs

```diff
@@ -6609,3 +6609,180 @@ fn sync_dry_run() -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn sync_script() -> Result<()> {
+    let context = TestContext::new_with_versions(&["3.8", "3.12"]);
+
+    let script = context.temp_dir.child("script.py");
+    script.write_str(indoc! { r#"
+        # /// script
+        # requires-python = ">=3.11"
+        # dependencies = [
+        #   "anyio",
+        # ]
+        # ///
+
+        import anyio
+       "#
+    })?;
+
+    let filters = context
+        .filters()
+        .into_iter()
+        .chain(vec![(
+            r"environments-v1/script-\w+",
+            "environments-v1/script-[HASH]",
```

Comment:
Should we add this filter to the test context, given that it's an unstable part of stderr for this workflow?